lib/htmlbuilder.js

```diff
@@ -25,10 +24,13 @@ module.exports.buildHtml = (auditData) => {
     var templateScript = Handlebars.compile(template);
 
     var html = templateScript({errors: errors});
+    var graph = '<html>Whatever</html>';
 
     fs.mkdirSync(path.join(__dirname, '/../../../output'), {
         recursive: true
     });
-    fs.writeFileSync(path.join(__dirname, '/../../../output/output.html'), html)
+    fs.writeFileSync(path.join(__dirname, '/../../../output/output.html'), html);
```

Comment:
Since the output dir is the same, do this:
```
const outputDir = path.join(__dirname, '/../../../output/);
```
Then you can do this in the writes:
```
fs.writeFileSync(outputDir + 'output.html'), html);
```
The one thing you'll need to check is if the path.join will automatically add the `/` at the end.  If it does, great, if not, the best thing is really to do this:
```
fs.writeFileSync(path.join(outputDir, 'output.htm'l), html);
```

Reply:
The latter might ultimately be better just because `path.join` takes into consideration the pathing strategy for the os it's on.  So it's guaranteed to work.